apigee_edge.module

```diff
@@ -1081,6 +1081,10 @@ function template_preprocess_app_credential(array &$variables) {
       ];
     }
     elseif ($def['value_type'] === 'status') {
+      // Check if expired.
+      if ($normalized['expiresAt'] !== -1 && \Drupal::time()->getRequestTime() - (int)($normalized['expiresAt'] / 1000) > 0) {
```

Comment:
A constant instead of -1 would be a little clearer.

Reply:
There's already a few places where -1 is used so I did not want to change that but I see how a constant could be better. We can do that in another fix. Thanks @kscheirer